ops/services/app_functions/report_stream_batched_publisher/_data.tf

```diff
@@ -0,0 +1,54 @@
+locals {
+  report_stream_token_secret_name = var.environment == "prod" ? "prod" : "test"
+}
+
+data "azurerm_resource_group" "rg_global" {
+  name = "prime-simple-report-management"
+}
+data "azurerm_key_vault" "sr_global" {
+  name                = "simple-report-global"
+  resource_group_name = data.azurerm_resource_group.rg_global.name
+}
+
+data "azurerm_key_vault_secret" "datahub_api_key" {
+  name         = "datahub-api-key-${local.report_stream_token_secret_name}"
+  key_vault_id = data.azurerm_key_vault.sr_global.id
+}
+
+data "azurerm_application_insights" "app" {
+  name                = "prime-simple-report-${var.environment}-insights"
+  resource_group_name = local.resource_group_name
+}
+
+data "azurerm_storage_account" "app" {
+  name                = "simplereport${var.environment}app"
+  resource_group_name = local.resource_group_name
+}
+
+data "azurerm_storage_account_sas" "sas" {
+  connection_string = data.azurerm_storage_account.app.primary_connection_string
+  https_only        = true
+  start             = "2021-09-01"
+  expiry            = "2022-12-31"
```

Comment:
How are we not going to forget about this ? More importantly if the team is entirely new people in 1.25 years, how are they going to know about this expiration date ?

Reply:
One idea is a slack reminder

Reply:
Created! 

![image](https://user-images.githubusercontent.com/18486822/134985160-44485a21-d2d8-4411-a119-072802cd23c9.png)